crates/starknet-server/tests/get_transaction_by_hash.rs

```diff
@@ -106,4 +113,79 @@ mod get_transaction_by_hash_integration_tests {
             FieldElement::from_hex_be(DECLARE_V2_TRANSACTION_HASH).unwrap()
         );
     }
+
+    #[tokio::test]
+    async fn get_deploy_account_transaction_by_hash_happy_path() {
+        let devnet = BackgroundDevnet::spawn().await.expect("Could not start Devnet");
+        let deploy_account_txn = BroadcastedDeployAccountTransaction {
+            max_fee: FieldElement::from_hex_be("0xde0b6b3a7640000").unwrap(),
+            signature: vec![
+                FieldElement::from_hex_be("0x1").unwrap(),
+                FieldElement::from_hex_be("0x1").unwrap(),
+            ],
+            nonce: FieldElement::from_hex_be("0x0").unwrap(),
+            class_hash: FieldElement::from_hex_be(
+                "0x7B3E05F48F0C69E4A65CE5E076A66271A527AFF2C34CE1083EC6E1526997A69", // UDC_CONTRACT_CLASS_HASH as string
+            )
+            .unwrap(),
+            contract_address_salt: FieldElement::from_hex_be("0x1").unwrap(),
+            constructor_calldata: vec![FieldElement::from_hex_be("0x1").unwrap()],
+        };
+
+        let deploy_transaction =
+            devnet.json_rpc_client.add_deploy_account_transaction(deploy_account_txn.clone()).await;
+
+        let result = devnet
+            .json_rpc_client
+            .get_transaction_by_hash(deploy_transaction.unwrap().transaction_hash)
+            .await
+            .unwrap();
+
+        if let starknet_rs_core::types::Transaction::DeployAccount(deploy) = result {
+            assert_eq!(
+                deploy.transaction_hash,
+                FieldElement::from_hex_be(DEPLOY_ACCOUNT_TRANSACTION_HASH).unwrap()
+            );
+        } else {
+            panic!();
```

Comment:
Here and in the other place, can you put some explanatory message argument to `panic!`?

Reply:
Fixed in 5d4615e1ff28bc1aa30d55ab33658090c1a14d0c

Reply:
I meant like `panic!("The value should be foo");`

Reply:
Fixed in 955242808ddb2d61160901c3828b72c56500c594